src/core/server/elasticsearch/legacy/cluster_client.ts

```diff
@@ -203,10 +205,20 @@ export class LegacyClusterClient implements ILegacyClusterClient {
     return new LegacyScopedClusterClient(
       this.callAsInternalUser,
       this.callAsCurrentUser,
-      filterHeaders(this.getHeaders(request), this.config.requestHeadersWhitelist)
+      filterHeaders(this.getHeaders(request), this.config.requestHeadersWhitelist),
+      this.getScopedScopedAuditor(request)
     );
   }
 
+  private getScopedScopedAuditor(request?: ScopeableRequest) {
+    if (request && isRealRequest(request)) {
```

Comment:
ES interface is quite relaxed due to BWC with the legacy platform, so we ignore empty & fakeRequests request objects. We should support alternative credential owners  from outside of Request context in https://github.com/elastic/kibana/issues/39430
until that all ES network interactions outside of RequestHandlerContext aren't written in the audition log.
@jportner, @thomheymann let me know if it's a problem I can try to find another solution.

Reply:
I think this is acceptable for now, thanks for the explanation.